gradle/libs.versions.toml

```diff
@@ -42,6 +42,7 @@ kotlin = "1.9.10"
 okhttp3 = "4.12.0"
 
 moshi = "1.15.0"
+## ⬆ = "1.15.1"
 
```

Comment:
What about this moshi update?

Reply:
Since it is unused accroding to the versions.properties file we skipped the update.

Reply:
pushed a new version where moshi is updated to 1.15.1